.github/workflows/CICD.yml

```diff
@@ -376,9 +374,15 @@ jobs:
       with:
         name: ${{ steps.debian-package.outputs.DPKG_NAME }}
         path: ${{ steps.debian-package.outputs.DPKG_PATH }}
+    - name: Check is release
```

Comment:
Maybe "Check for release"? Or "Check for release version tag"?

Reply:
I agree "Check is release" sounds weird if one reads it like regular English (which is a reasonable thing to do) rather than dry systematic specification of a build step, which is what it currently is.

I'll change to "Check for release",